src/datasets/other/attitudes_daily/reference.md

```diff
@@ -0,0 +1,81 @@
+# Attitudes Daily
+
+<!-- toc -->
+
+# Introduction
+
+{{#include ./intro.md}}
+
+# Data Reference
+
+The DAS shows the user the following four statements, all having the same possible responses of "Agree", "Disagree" or "Neutral or not sure".
+Each question has an identifier in the `attitudes_daily` table as the `question_key`field (shown in parentheses after each statement below)
+
+* The internet is open and accessible to all (`internet_accessible`)
+* I trust Firefox to help me with my online privacy (`trust_firefox`)
+* All the sites I’ve visited recently have worked; none of them seem broken (`sites_work`)
+* Using the internet helped me meet my goals today (`met_goals`)
+
+User responses are processed and mapped to their `question_key`'s in [this script](https://github.com/mozilla/bigquery-etl/blob/master/templates/telemetry_derived/surveygizmo_daily_attitudes/import_responses.py), creating a table `moz-fx-data-shared-prod.external.survey_gizmo_daily_attitudes` which is subsequently joined to `clients_daily`.
+
+
+
+
+## Example Queries
+
+#### Distribution of user responses to "The internet is open and accessible to all" over time
+
+```sql
+SELECT 
+    submission_date,
+    value,
+    count(DISTINCT client_id) as n
+FROM `moz-fx-data-shared-prod.telemetry_derived.attitudes_daily_v1`
+WHERE 
+    question_key = 'internet_accessible'
+    AND value IS NOT NULL
+GROUP BY 1, 2
+```
+
+<iframe src="https://sql.telemetry.mozilla.org/embed/query/65079/visualization/165757?api_key=oSjO27fGmpCsnBXBEhaysRVrLZpX1SKMCpYcxA5h&" width="100%" height="530" frameborder="0" scrolling="no"></iframe> 
+
+#### Distribution of responses to "I was able to get what I wanted from using the Internet today" by default search engine.
+
+```sql
+SELECT 
+    value,
+    CASE
+      WHEN STARTS_WITH(default_search_engine, 'google') THEN 'Google'
+      WHEN STARTS_WITH(default_search_engine, 'ddg') THEN 'DuckDuckGo'
+      WHEN STARTS_WITH(default_search_engine, 'bing') THEN 'Bing'
+      ELSE 'Other' 
+    END AS search_engine,
+   count(*) AS n
+FROM 
+    `moz-fx-data-shared-prod.telemetry_derived.attitudes_daily_v1`
+WHERE
+    question_key = 'met_goals'
+    AND value IS NOT NULL
+    AND submission_date > DATE_SUB(CURRENT_DATE, INTERVAL '7' DAY)
+GROUP BY 1, 2
+```
+
+<iframe src="https://sql.telemetry.mozilla.org/embed/query/63957/visualization/163467?api_key=gzCopzybDta4t3JGkx2urB9MQa67akAehJUybVdW&" width="100%" height="570" frameborder="0" scrolling="no"></iframe>
+
+## Scheduling
+
+This dataset is updated daily via the
+[telemetry-airflow](https://github.com/mozilla/telemetry-airflow) infrastructure.
+The job runs as part of the [`attitudes_daily` DAG](https://github.com/mozilla/telemetry-airflow/blob/master/dags/attitudes_daily.py).
```

Comment:
If we add a link to https://workflow.telemetry.mozilla.org/tree?dag_id=attitudes_daily folks will be able to see actual  task run status as well

Reply:
👍